Makefile

```diff
@@ -111,12 +108,16 @@ Documentation/telemeter_query: manifests/0000_50_cluster_monitoring_operator_04-
 ##############
 
 .PHONY: format
-format: go-fmt shellcheck
+format: go-fmt shellcheck jsonnet-fmt
```

Comment:
should `test-rules` be also part of this rule?

Reply:
I changed name to `check-rules` and moved it to formatting section